docs/tutorials/mint-farm-yusdbtc.md

```diff
@@ -0,0 +1,150 @@
+
+`Destination` should be your Metamask wallet address, this is where your renBTC will be sent to.
+
+Also check the cost (`RenVM fee`, `Bitcoin Network fee`) and the amount of renBTC you will be receiving.
+
+Once you have done all of this, you can then go ahead and click `confirm`.
+
+![confirm](/docs/tutorials/renBTC_confirm.png)
+
+A one-time deposit address is generated for you to send your BTC to. Send the exact amount of BTC you are wrapping to that address, in a single transaction.
+
+![deposit](/docs/tutorials/renBTC_deposit.png)
+
+* Your transaction requires 6confirmations(can take up to 1hour) before renBTC is sent to your wallet 
+* You will need to accept & sign the transaction that sends renBTC to your wallet.
+
+
+
+## Step 2: Minting uUSDrBTC with renBTC
+
+Minting uUSDrBTC tokens is easy with the [EMP Tools](http://tools.umaproject.org/),
+UMA's UI for ExpiringMultiParty contracts (EMP) which is what the uUSDrBTC token is
+minted from.
+
+Navigate to http://tools.umaproject.org/ and click the "Connect" button at the
+top right corner to connect your MetaMask wallet to the dapp.
+
+![connect](/docs/tutorials/emp_connect.png)
+
+Then, select the `uUSDrBTC` contract from the dropdown immediately underneath the
+header (above the tabs).
+
+![select EMP](/docs/tutorials/uUSD_selectemp.png)
+
+Select the "Manage Position" tab.
+
+![manage position tab](/docs/tutorials/uUSD_manage-positionemp.png)
+
+Scroll down to the "Actions" dropdown and ensure that "Create" is selected.
+
+![position actions dropdown](/docs/tutorials/emp_actions.png)
+
+At this point, make sure you read the text and confirm that:
+
+1. You will be minting at a collateralization ratio above the global
+   collateralization ratio (GCR);
+2. You will be minting the minimum required number of tokens (100 for `uUSDrBTC`),
+   and;
+3. You will keep your position collateralized above the minimum required
+   collateralization ratio (CR) or else risk getting liquidated (the required CR
+   is 125% for `uUSDrBTC`).
+
+If you fully understand the above, and have decided on an appropriate amount of
+(1) collateral to supply and (2) tokens to mint, check to make sure you have
+enough collateral for minting.
+
+![token minting form](/docs/tutorials/uUSD_mintingform.png)
+
+In the first field, fill in the number of tokens you want to mint.
+
+In the second field, fill in the amount of collateral you want to supply.
+
+If these values are appropriate, you can then click the "Create" button which
+will trigger a transaction with MetaMask. Once that is confirmed and the
+transaction is mined, you shall have your `uUSDrBTC` tokens!
+
+## Step 3: Farming uUSDrBTC on Balancer
+
+:::danger
+
+Supplying _only_ `USDC` to the pool is very dangeorus and is not recommended,
+because you are essentially buying `uUSDrBTC` on the open market. You should avoid
+this because `uUSDrBTC` will be worth
+$1 by the time of expiry and you will lose
+money if you are buying `uUSDrBTC` above $1.
+
```

Comment:
Change the above warning to say instead:

Supplying _only_ 'USDC' to the pool will result in auto-purchasing uUSD at the current price.  This means you are market-buying uUSDrBTC, so be mindful of the trading price of uUSDrBTC before doing this.